src/sentry/integrations/slack/metrics.py

```diff
@@ -17,28 +17,6 @@
 SLACK_NOTIFY_RECIPIENT_SUCCESS_DATADOG_METRIC = "sentry.integrations.slack.notify_recipient.success"
 SLACK_NOTIFY_RECIPIENT_FAILURE_DATADOG_METRIC = "sentry.integrations.slack.notify_recipient.failure"
 
-# Bot commands
```

Comment:
this is a different metrics thing btw. might want to structure those automated PRs based on codeowners

Reply:
yeah tbf I just did `| grep metrics` 🤷